installation/linux.md

```diff
@@ -403,12 +403,13 @@ sudo useradd openhab tty
 ```
 
 Additionally it's needed to allow the java environment to access the serial port of the connected peripheral.
-Therefor the following setting has to be added/adapted on your system in either `/etc/default/openhab2` or `/usr/share/openhab2/runtime/bin/setenv`:
+Therefor the following setting has to be added/adapted on your system in file `/etc/default/openhab2`:
```

Comment:
Sorry to be picky, but this word should have been "Therefore", which means "consequently". Therefor is a connecting word that basically means "for it". Didn't see it in the first read because they're often confused. Only grammatical comment I have, the rest reads fine to me!